admin_manual/release_notes.rst

```diff
@@ -12,6 +13,61 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.4_release_notes_label:
+
+Changes in 10.0.4
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.4 that need your attention:
+
+**The full ownCloud Server 10.0.4 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* **More granular sharing restrictions**
+The option "Restrict users to only share with users in their groups" in the admin sharing settings forces users to only be able to share with groups they are member of while simultaneously prohibiting to share with single users that do not belong to any of the users' groups. To make this more granular we split this option into two and added "Restrict users to only share with groups they are member of" which now differentiates between users and groups making it possible to force users to be able to share with all users of an installation but only with groups they are member of and vice versa. Please check if the new possibilities better suit your needs.
+
+- **Configurable solution for indistinguishable user display names**
+In the ownCloud sharing dialog users are displayed according to their display name. As users can choose their display name in self-service (can be disabled in config.php) and display names are not unique it is possible that a user can't distinguish sharing results. To cover this case the displayed user identifiers are now configurable. Administrators can configure to additionally display mail addresses or user ids. Please check if this new feature better suits your needs.
+
+- **Added "occ files:scan" repair mode to repair filecache inconsistencies**
+We recommend to use this command when directed to do so in the upgrade process (TODO more info)
+Please check the ownCloud documentation for more information. **LINK (https://github.com/owncloud/documentation/issues/3446)**
+
+- **Detailed mode for "occ security:routes"**
+Administrators can use the output of this command when using a network firewall to check the appropriateness of configured rules or to get assistance when setting up.
+
+- **Added mode of operations to differentiate between single-instance or clustered setup**
+As ownCloud needs to behave differently when operating in a clustered setup we added a config.php option ``'operation.mode' => 'single-instance' or 'clustered-instance'``. 
```

Comment:
For utmost clarity, I'd write this a little differently:

As ownCloud needs to behave differently when operating in a clustered setup to a single instance setup, we added a new `config.php` option `operation.mode`. It can take one of two values: `single-instance` and `clustered-instance`.
For example:

```
'operation.mode' => 'single-instance',
```

Reply:
clustered setup to a single instance setup --> clustered setup versus a single instance setup

we added a new ``config.php`` option ``operation.mode`` --> a new ``config.php`` option ``operation.mode`` has been added

Reply:
changed